level3/[level3] 이도형/java-racing-car/src/main/java/Model/Car.java

```diff
@@ -0,0 +1,38 @@
+package Model;
+
+
+import camp.nextstep.edu.missionutils.Randoms;
+import org.junit.jupiter.api.MethodOrderer;
+
+public class Car {
+    private final String name;
+    private int position = 0;
+
+    public Car(String name) {
+        this.name = name;
+    }
+
+    // 추가 기능 구현
+
+    public void getRandomNumber(){
+        int randomNumber = Randoms.pickNumberInRange(1, 10);
+        if(randomNumber>4){
+            goForward();
+        }
+    }
+    public void goForward(){
+        this.position += 1;
+    }
+
+    public int getPosition(){
+        return this.position;
+    }
+
+    public String showPosition(){
+        return "-".repeat(this.position);
+    }
```

Comment:
show라는 동사는 보여준다는 의미, 출력할 것 같은 이름같습니다.
아래와 같은 이름이 좋아보입니다
```suggestion
    public String getPositionMarker(){
        return "-".repeat(this.position);
    }
```